packages/antd/src/widgets/SubmitButton/SubmitButton.js

```diff
@@ -5,7 +5,7 @@ const { getSubmitButtonOptions } = utils;
 export default ({ uiSchema }) => {
   const { submitText, norender, props: submitButtonProps }= getSubmitButtonOptions(uiSchema);
   if (norender) {return null;}
-  return (<Button  type="submit" {...submitButtonProps}>
+  return (<Button  htmlType="submit" {...submitButtonProps}>
```

Comment:
Minor little thing... there seems to be an extra space between `Button` and `htmlType` that you are welcome to remove even though it was there before

Reply:
It's all done. :)

Reply:
It look like you are going to have to update the test snapshots as a result of this change

Reply:
I have updated snapshots. You can try to trigger workflow.